RecoEgamma/EgammaIsolationAlgos/plugins/EgammaHcalPFClusterIsolationRemapper.cc

```diff
@@ -0,0 +1,63 @@
+#include "RecoEgamma/EgammaIsolationAlgos/plugins/EgammaHcalPFClusterIsolationRemapper.h"
```

Comment:
what's the reason this is needed?
If I change ``` s/HcalPF/EcalPF/g ```, this becomes identical to EgammaEcalPFClusterIsolationRemapper

please cleanup